doc/employees.md

```diff
@@ -1 +1,605 @@
+
+Request Payload :
+```
+{
+    "employee_kode": "20200301",
+    "employee_nama": "ferry",
+    "stall_code": "Heavy Repair"
+}
+```
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5fbf4995-b4ac-4a2e-9cad-4021cf944ef7",
+        "employee_kode": "20200301",
+        "employee_nama": "ferry",
+        "stall_code": "Heavy Repair",
+        "jabatan_id": "5d2c04c9-fffc-4175-ac06-148dcf944ef7",
+        "company_id": "5e1befd2-ab04-49c2-addd-5b25c0a8141e",
+        "company_code": "cdc-01",
+        "created" : "2020-12-03 09:44:02",
+        "modified" : "2020-12-03 09:44:02"
+    }
+}
+```
+
+## <a name="view"></a>View Data Employee
+
+### Endpoint
+GET /employees/:id
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | *
+Accept | application/json
+Email | alamat@emailcdc.com
+
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+404 | Not Found | User not found in database  
+500 | Internal Server Error | some un-handle error in server 
+200 | OK | OK
+
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5fbf4995-b4ac-4a2e-9cad-4021cf944ef7",
+        "employee_kode": "20200301",
+        "employee_nama": "ferry",
+        "stall_code": "Heavy Repair",
+        "jabatan_id": "5d2c04c9-fffc-4175-ac06-148dcf944ef7",
+        "company_id": "5e1befd2-ab04-49c2-addd-5b25c0a8141e",
+        "company_code": "cdc-01",
+        "created" : "2020-12-03 09:44:02",
+        "modified" : "2020-12-03 09:44:02"
+    }
+}
+```
+
+### Scenario Test
+
+#### Case : Positive Case
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5fbf4995-b4ac-4a2e-9cad-4021cf944ef7",
+        "employee_kode": "20200301",
+        "employee_nama": "ferry",
+        "stall_code": "Heavy Repair",
+        "jabatan_id": "5d2c04c9-fffc-4175-ac06-148dcf944ef7",
+        "company_id": "5e1befd2-ab04-49c2-addd-5b25c0a8141e",
+        "company_code": "cdc-01",
+        "created" : "2020-12-03 09:44:02",
+        "modified" : "2020-12-03 09:44:02"
+    }
+}
+```
+
+# <a name="edit"></a>Edit Data Employee
+
+### Endpoint
+PUT /employees/:id
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | *
+Accept | application/json
+Email | alamat@emailcdc.com
+
+### Request Payloads
+Name | Type | Example Value
+--- | --- | ---
+employee_kode | string | 200301  
+employee_nama | string | ferry setiawan
+stall_code | string | Reguler
+```
+{
+    "employee_kode": "200301",
+    "employee_nama": "ferry setiawan",
+    "stall_code": "Reguler"
+}
+```
+
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+400 | Bad Request | Bad request payload  
+404 | Not Found | User not found in database  
+500 | Internal Server Error | some un-handle error in server 
+200 | OK | OK
+```
+{
```

Comment:
why is it different from the one below? for negative response example, you should use bad request, not found or forbidden response